src/accordionpane/createAccordionPaneElement.ts

```diff
@@ -8,8 +8,20 @@ export default function createAccordionPaneElement(): CustomElementDescriptor {
 	return {
 		tagName: 'dojo-accordionpane',
 		widgetConstructor: AccordionPane,
-		attributes: [],
-		properties: [],
-		events: []
+		properties: [
```

Comment:
This was an omission on my part with the AccordionPane PR.

Reply:
Why is it part of a tooltip PR?

Reply:
Removed. Opened #364 to track adding this and the AccordionPane README.